Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -195,7 +195,7 @@ def check_for_other_package_managers
       end
 
       def check_ruby_version
-        ruby_version = "2.0"
+        ruby_version = MacOS.version >= "10.13" ? "2.3" : "2.0"
```

Comment:
I don't want to go back to supporting different Ruby versions in different places. CC @mistydemeo how's the Ruby `2.3.3p222` stuff looking? CC @alyssais @xu-cheng in case either of you have been working on the portable Ruby recently, either.

Reply:
I don't think there is any problem for portable ruby 2.3. except that you have to build it on your own (I don't have access to 10.5 VM).